testcases/misc_tests/p11sak_test.sh

```diff
@@ -353,6 +369,81 @@ else
 	status=1
 fi
 
+if [[ -n $( pkcsconf -m -c $SLOT | grep CKM_AES_XTS_KEY_GEN) ]]; then
+	# check AES-XTS 128
+	grep -q 'p11sak-aes-xts-128' $P11SAK_AES_XTS_PRE
+	rc=$?
+	if [ $rc = 0 ]; then
+		echo "* TESTCASE generate-key aes-xts-128 PASS Generated random AES-XTS 128 key"
+	else
+		echo "* TESTCASE generate-key aes-xts-128 FAIL Failed to generate AES-XTS 128 key"
+		status=1
+	fi
+	grep -v -q 'p11sak-aes-xts-128' $P11SAK_AES_XTS_POST
+	rc=$?
+	if [ $rc = 0 ]; then
+		echo "* TESTCASE remove-key aes-xts-128 PASS Deleted generated AES-XTS 128 key"
+	else
+		echo "* TESTCASE remove-key aes-xts-128 FAIL Failed to delete generated AES-XTS 128 key"
+		status=1
+	fi
+
+	# check AES-XTS 256
+	grep -q 'p11sak-aes-xts-256' $P11SAK_AES_XTS_PRE
+	rc=$?
+	if [ $rc = 0 ]; then
+		echo "* TESTCASE generate-key aes-xts-256 PASS Generated random AES-XTS 256 key"
+	else
+		echo "* TESTCASE generate-key aes-xts-256 FAIL Failed to generate AES-XTS 256 key"
+		status=1
+	fi
+	grep -v -q 'p11sak-aes-xts-256' $P11SAK_AES_XTS_POST
+	rc=$?
+	if [ $rc = 0 ]; then
+		echo "* TESTCASE remove-key aes-xts-256 PASS Deleted generated AES-XTS 256 key"
+	else
+		echo "* TESTCASE remove-key aes-xts-256 FAIL Failed to delete generated AES-XTS 256 key"
+		status=1
+	fi
+	
+	# CK_BBOOL
+	if [[ $(grep -A 69 'p11sak-aes-xts-128' $P11SAK_AES_XTS_LONG | grep -c 'CKA_IBM_PROTKEY_EXTRACTABLE: CK_FALSE') == "2" ]]; then
+		echo "* TESTCASE list-key aes-xts PASS Listed random aes-xts public keys CK_BBOOL attribute"
+	else
+		echo "* TESTCASE list-key aes-xts FAIL Failed to list aes-xts public keys CK_BBOOL attribute"
+		status=1
+	fi
+	# CK_ULONG
+	if [[ $(grep -A 69 'p11sak-aes-xts-128' $P11SAK_AES_XTS_LONG | grep -c 'CKA_MODULUS_BITS:') == "0" ]]; then
+		echo "* TESTCASE list-key aes-xts PASS Listed random aes-xts public keys CK_ULONG attribute"
+	else
+		echo "* TESTCASE list-key aes-xts FAIL Failed to list aes-xts public keys CK_ULONG attribute"
+		status=1
+	fi
+	# CK_BYTE
+	if [[ $(grep -A 69 'p11sak-aes-xts-128' $P11SAK_AES_XTS_LONG | grep -c 'CKA_MODULUS:') == "0" ]]; then
+		echo "* TESTCASE list-key aes-xts PASS Listed random aes-xts public keys CK_BYTE attribute"
+	else
+		echo "* TESTCASE list-key aes-xts FAIL Failed to list aes-xts public keys CK_BYTE attribute"
+		status=1
+	fi
+	# URI
+	if [[ $(grep -A 69 'p11sak-aes-xts-128' $P11SAK_AES_XTS_LONG | grep -c 'URI: pkcs11:.*type=secret-key') == "2" ]]; then
+		echo "* TESTCASE list-key aes-xts PASS list aes-xts key pkcs#11 URI"
+	else
+		echo "* TESTCASE list-key aes-xts FAIL list aes-xts key pkcs#11 URI"
+		status=1
+	fi
+else
+	echo "* TESTCASE generate-key aes-xst-128 SKIP Generated random AES-XTS 128 key"
+	echo "* TESTCASE remove-key aes-xts-128 SKIP Deleted generated AES-TS 128 key"
+	echo "* TESTCASE generate-key aes-xst-256 SKIP Generated random AES-XTS 128 key"
+	echo "* TESTCASE remove-key aes-xts-256 SKIP Deleted generated AES-TS 128 key"
```

Comment:
Typos: 
AES-TS -> AES-XTS
And in lines 440 + 441: use both times 256 (see end of line: 128)

Reply:
will fix